eZ/Bundle/EzPublishRestBundle/Tests/Functional/TestCase.php

```diff
@@ -88,7 +90,9 @@ protected function setUp()
     public function sendHttpRequest(HttpRequest $request)
     {
         $response = new HttpResponse();
-        $this->httpClient->send($request, $response);
+
+        $request = RequestConverter::psr7($request);
+        ResponseConverter::copy(ResponseConverter::buzz($this->httpClient->sendRequest($request, [])), $response);
```

Comment:
Two issues I have doubts about:
1) Why do you need to copy `Response`? `::buzz` method should already give you correct object to return.
2) Second argument of `sendRequest` is optional and defaults to `[]` so it's not needed here.

I see the body of this method as:
```php
    public function sendHttpRequest(HttpRequest $request)
    {
        $request = RequestConverter::psr7($request);

        return ResponseConverter::buzz($this->httpClient->sendRequest($request));
    }
```

However, I'm not very familiar with Buzz, so maybe I'm missing something?